gemfirexd/prebuild/src/main/resources/com/pivotal/gemfirexd/internal/impl/jdbc/metadata.properties

```diff
@@ -132,18 +132,22 @@ getProcedures40=\
 #
 getSchemas=\
         SELECT SCHEMANAME AS TABLE_SCHEM, \
-          CAST(NULL AS VARCHAR(128)) AS TABLE_CATALOG \
-          FROM SYS.SYSSCHEMAS \
-	  WHERE ((1=1) OR ? IS NOT NULL) \
-          AND SCHEMANAME LIKE ? \
-\
-UNION \
-\
-	SELECT DISTINCT SCHEMANAME AS TABLE_SCHEM, \
-		CAST(NULL AS VARCHAR(128)) AS TABLE_CATALOG \
-	FROM SYS.HIVETABLES \
-	WHERE SCHEMANAME LIKE ? \
-ORDER BY TABLE_SCHEM
+            CAST(NULL AS VARCHAR(128)) AS TABLE_CATALOG \
+        FROM SYS.SYSSCHEMAS \
+        WHERE ((1=1) OR ? IS NOT NULL) \
+            AND SCHEMANAME LIKE ? \
+            AND SCHEMANAME NOT IN ('NULLID', 'SNAPPY_HIVE_METASTORE','SQLJ','SYSCAT', \
+                'SYSCS_DIAG','SYSCS_UTIL','SYSFUN','SYSIBM','SYSPROC','SYSSTAT') \
+        \
+        UNION \
+        \
+        SELECT DISTINCT SCHEMANAME AS TABLE_SCHEM, \
+            CAST(NULL AS VARCHAR(128)) AS TABLE_CATALOG \
+        FROM SYS.HIVETABLES \
+        WHERE SCHEMANAME LIKE ? \
+            AND SCHEMANAME NOT IN ('NULLID', 'SNAPPY_HIVE_METASTORE','SQLJ','SYSCAT', \
```

Comment:
Hi Neeraj,

I have added check for internal schema names in SYS.HIVETABLES query also. Is it really required ?